docker/compose/indexer-postgres/docker-compose.yaml

```diff
@@ -0,0 +1,71 @@
+version: "3.8"
+services:
+  db:
+    container_name: postgres
+    image: postgres
+    restart: always
+    volumes:
+      - aptos-indexer-db:/var/lib/postgresql/data
+    env_file:
+      - ./.env
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
```

Comment:
I feel mixed about it, but I'd be curious how you feel about making these use the defaults if the env isn't specified... but I don't know if env_file will fail if it doesn't exist.

I remember a really horrible experience with yarn and missing env vars being horrible to decipher :P.